src/Controller/Comment.php

```diff
@@ -1,15 +1,14 @@
 <?php
-Namespace MOOP\Controller;
+namespace MOOP\Controller;
 
-Use PDO;
+use MOOP\Model\Comment as CommentModel;
 
 class Comment {
+	
+	private $model;
```

Comment:
A note about the use of private...if you're 100% sure you'll never extend the object, feel free to use it. Otherwise, you'll find it screws you.